pom.xml

```diff
@@ -40,23 +41,23 @@ THE SOFTWARE.
 
   <name>Maven Integration plugin</name>
   <description>This plug-in provides, for better and for worse, a deep integration of Jenkins and Maven: Automatic triggers between projects depending on SNAPSHOTs, automated configuration of various Jenkins publishers (Junit, ...).</description>
-  <url>https://github.com/jenkinsci/maven-plugin</url>
+  <url>https://github.com/jenkinsci/${project.artifactId}</url>
 
   <properties>
     <revision>3.17</revision>
     <changelist>-SNAPSHOT</changelist>
-    <gitHubRepo>jenkinsci/maven-plugin</gitHubRepo>
-    <jenkins.version>2.289.1</jenkins.version>
+    <gitHubRepo>jenkinsci/${project.artifactId}</gitHubRepo>
+    <jenkins.version>2.321</jenkins.version>
```

Comment:
when LTS will be release?

Reply:
Planned 2022-03-09. I do not see the harm in publishing this now to have a clean release for current weekly users (see discussion in https://issues.jenkins.io/browse/JENKINS-67102 which is suspected to pertain to the Guava bump in core).

Reply:
ok sad to see there are some users facing issue. 
Please add directly such jira link so it's known there are users having issues. 
Sorry but it's hard to read all notifications and find the important ones in the middle of all the noise.

Reply:
As mentioned in the original PR, not confirmed: https://github.com/jenkinsci/maven-plugin/pull/230#issuecomment-972609528

Reply:
the issue seems to be closed and not anymore an issue as reported by the OP here https://issues.jenkins.io/browse/JENKINS-67102?focusedCommentId=416208&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-416208
Anyway as it looks be very urgent to do it now:
- can you please update the PR title so it will be clear in the changelog this new version requires a new core version **not yet LTS**
- I will appreciate your help to explain if any users ask/complains the reason about such change and why they cannot use the last release without upgrading to a non LTS version

Reply:
> the issue seems to be closed and not anymore an issue

Yes, TBD.

Reply:
> update the PR title

sure

> explain if any users ask/complains the reason about such change and why they cannot use the last release without upgrading to a non LTS version

No explanation is necessary. “Recent versions of this plugin require recent versions of Jenkins.” Happens pretty routinely.

Reply:
@jmMeessen wrote a [blogpost](https://www.jenkins.io/blog/2022/02/10/last-plugin-version-not-installable/) that describes the situation for LTS users, in case that helps with the explanation.

Reply:
sure but 
```Note that this story is a strong reminder to keep your Jenkins system as current as possible if you don’t want to miss out on new features and important bug fixes.```
we need to understand not everybody will be happy to have to upgrade every weeks with all the possible bugs every week. Jenkins can be critical so upgrade every week is risky for some organisations...

maybe LTS line should be updated more often?

Reply:
> we need to understand not everybody will be happy to have to upgrade every weeks with all the possible bugs every week. Jenkins can be critical so upgrade every week is risky for some organisations...

I agree that many orgs choose the LTS line so that they upgrade less frequently.  The choice to upgrade Jenkins core less frequently means that I also am choosing that sometimes I won't be able to update plugins.  Can you help me understand the concern more fully?

> maybe LTS line should be updated more often?

I don't understand why this scenario would prompt us to change the LTS release frequency.  Yes, we could switch to a different LTS timeline, like "new LTS baseline every month", but wouldn't that only change the length of the time window where this condition occurs?